code/modules/projectiles/guns/hoboguns.dm

```diff
@@ -441,13 +350,10 @@
 	scope_x_offset = 11
 	scope_y_offset = 14
 	fire_sound = 'sound/f13weapons/lasmusket_fire.ogg'
-	cock_sound = 'sound/f13weapons/lasmusket_crank.ogg'
+	pump_sound = 'sound/f13weapons/lasmusket_crank.ogg'
```

Comment:
Is `pump_sound` still a var? Should be `cock_sound`. Though if it compiles, it might be fine.